src/components/Searchbox/Searchbox.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { User } from 'types/userTypes';
+import { setUserListFilter } from 'store/slices/userSlice';
+
+type FilteredUser = User[] | [];
+
+const Searchbox = () => {
+  const dispatch = useDispatch();
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const userList = useSelector((state: any) => state.userSlice.userList);
+  const [inputValue, setInputValue] = useState('');
+
+  const onChangeHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
`e: React.ChangeEvent<HTMLInputElement>` required type that avoids target.value issues